test/test_ops.py

```diff
@@ -260,6 +259,7 @@ def __new__(cls, name, variant_test_name=""):
             # AllowedOpInfoEntry('norm', 'inf'),
             # AllowedOpInfoEntry('max', 'reduction_with_dim'),
             # AllowedOpInfoEntry('min', 'reduction_with_dim'),
+            # AllowedOpInfoEntry('log_softmax'),  // scalar tensor dim 0
```

Comment:
It sounds like this is no longer necessary after https://github.com/pytorch/xla/issues/3061#issuecomment-889607863?

I'm torn, because it seems better for the xla-special-casing to happen in `pytorch/xla` instead of core (i.e. in that PR [here](https://github.com/pytorch/pytorch/pull/62077/files#diff-18a0c3e7cd07089b34760028c387d39de1aa393172f1d5f123692106b3a3d144R3686)). But the whole "xla doesn't treat 0-dim tensors as 1-dim" probably affects a bunch of ops, not just `log_softmax`, so that's a more general bandaid fix.

Reply:
sounds good. It seems like we no longer need this pr.